azure-pipelines.yml

```diff
@@ -1,26 +1,27 @@
 trigger:
   branches:
     include:
-    - personal/yankhuan/converter-vsc-extension
     - main
-  tags:
-    include:
-    - v*
 
-jobs:
-- job: FHIR_Converter_CI_Tests
-  displayName: FHIR-Converter CI Tests
+stages: 
+- stage: FhirConverter
+  jobs:
+  - job: Build
+    displayName: FHIR Converter Build -
 
-  strategy:
-    matrix:
-      windows:
-        imageName: 'windows-2019'
+    strategy:
+      matrix:
+        Windows:
+          imageName: 'windows-latest'
+        MacOS:
+          imageName: "macos-latest"
```

Comment:
Does this mean that we are going to build the extension for macos?

Reply:
Yes, we can run different platform pipelines in parallel. 
Now we will test the function of packaging in the macos. After the engine is ready, we will run `npm run test` to run the tests by changing this condition `condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))`

Reply:
Got it. Thanks!